airflow/www/views.py

```diff
@@ -3251,9 +3231,9 @@ def duration(self, dag_id: str, session: Session = NEW_SESSION):
         cum_chart.buildcontent()
         s_index = cum_chart.htmlcontent.rfind("});")
         cum_chart.htmlcontent = (
-            cum_chart.htmlcontent[:s_index]
-            + "$( document ).trigger('chartload')"
-            + cum_chart.htmlcontent[s_index:]
+            f"{cum_chart.htmlcontent[:s_index]}"
+            f"$( document ).trigger('chartload')"
```

Comment:
```suggestion
            "$( document ).trigger('chartload')"
```

Doesn't need to be an fstring